cli/src/index.ts

```diff
@@ -325,13 +324,11 @@ export const withCommonOptions = <C extends RootCommand>(command: C) =>
     })
     .group([jiraHostOptionKey, jiraProtocolOptionKey], "Common Required:");
 
-// eslint-disable-next-line @typescript-eslint/explicit-function-return-type
 export const withAuthenticationOptions = <C extends RootCommand>(command: C) =>
   withCommonOptions(command)
     .group([jiraConsumerKeyOptionKey, jiraConsumerSecretOptionKey], "Auth:")
     .demandOption([jiraHostOptionKey, jiraConsumerSecretOptionKey]);
 
-// eslint-disable-next-line @typescript-eslint/explicit-function-return-type
```

Comment:
Are these removed because the AD linter config doesn't flag them or because they were unneeded per the original configuration?

Reply:
The latter - `--report-unused-disable-directives` spotted they were always unnecessary. Two separate commits: https://github.com/agiledigital/jiralint/pull/226/commits